src/test/java/org/mvel2/tests/BaseMvelTestCase.java

```diff
@@ -0,0 +1,19 @@
+package org.mvel2.tests;
+
+import junit.framework.TestCase;
+
+import static org.mvel2.tests.BaseMvelTest.cleanUpConfigurations;
+
+/**
+ * Base class for Junit 3 TestCase.
+ */
+public class BaseMvelTestCase extends TestCase {
```

Comment:
In order to deal with JUnit 3 style tests and JUnit 4 style tests, I introduced `BaseMvelTestCase` and `BaseMvelTest` accordingly. I may reorganize all tests to meet JUnit 4 (or 5?), but it would be another JIRA.